RussianFilter/sections/specific.txt

```diff
@@ -18,6 +18,8 @@ Ly9hbi55YW5kZXgucnUv
 ! adbetnet domains
 ||cdn.*/libs/b.js$third-party,domain=nnm-club.lib|nnm-club.me|nnm-club.tv|nnmclub.to|seasonvar.ru|mintmanga.com|realmadrid.ru
 !
+adultmult.xyz##a[href^="https://bp2.infiplay.com/"]
+adultmult.xyz##a[href^="https://www.vsemayki.ru/"]
```

Comment:
Last comment was only for rule with ".vsemayk"

Reply:
Check if existing adultmult.net rules are works here